fix sndio detection: gcc complains that "control reaches end of non-void
function".

Index: tests.c
--- tests.c.orig
+++ tests.c
@@ -526,6 +526,7 @@ main(void)
 	hdl = sio_open(SIO_DEVANY, SIO_PLAY, 1);
 	sio_flush(hdl);
 	sio_close(hdl);
+	return 0;
 }
 #endif /* TEST_SIO_FLUSH */
 #if TEST_SCAN_SCALED
@@ -950,5 +951,6 @@ main(void)
 	struct sio_hdl *hdl;
 	hdl = sio_open(SIO_DEVANY, SIO_PLAY, 1);
 	sio_close(hdl);
+	return 0;
 }
 #endif /* TEST_LIB_SNDIO */
